CMakeLists.txt

```diff
@@ -11,14 +11,15 @@ endif()
 project("openscap")
 set(OPENSCAP_VERSION_MAJOR "1")
 set(OPENSCAP_VERSION_MINOR "3")
-#TODO: Remove this before final 1.3.0
-#FIXME: Windows Installer allows only integers in version name.
-if(WIN32)
-	set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION_PATCH "0")
+set(OPENSCAP_VERSION_SUFFIX "alpha2")
+
+if(OPENSCAP_VERSION_SUFFIX AND NOT WIN32)
+	set(OPENSCAP_VERSION "${OPENSCAP_VERSION_MAJOR}.${OPENSCAP_VERSION_MINOR}.${OPENSCAP_VERSION_PATCH}.${OPENSCAP_VERSION_SUFFIX}")
```

Comment:
NACK, this is not how this should be handled, same version on all platforms!

Reply:
@mpreisler can we then change `_alpha2` to `.9999`? Can't use a string in versioning for some platforms e.g. `windows`.

Reply:
I think this applies only to the Windows installer. In other places we could use the full string.

Reply:
So, recommendations for path forward?

Reply:
@redhatrises actually shouldn't the suffix be added after a dash instead of a dot?

Reply:
Well, it looks like rpm won't like the `-` in the version.

Reply:
> @redhatrises actually shouldn't the suffix be added after a dash instead of a dot? Would windows build accept strings after the dash?

@yuumasato no. http://wixtoolset.org/documentation/manual/v3/xsd/wix/simple_type_versiontype.html

Reply:
> NACK, this is not how this should be handled, same version on all platforms!

It seems inevitable if this is a Windows characteristic.

Idea for the windows installer:
How about passing as the version`1.3.0` to `msi` installer, but 
- Set up `oscap -v` to show "correct" version, i.e `1.3.0_alpha2`,
- Show a warning during installation saying that this installs an alpha version,
- And also set CPACK_PACKAGE_NAME "OpenSCAP Alpha2"

Reply:
Changing CPACK_PACKAGE_NAME is not a good solution, because the CPACK_PACKAGE_VERSION is appended to the package name anyway.
I think for now we can survive that the information will not be displayed in the Windows Installer.